### PR TITLE
Election polling station progress bars

### LINKED ISSUE #242: Add polling station progress bars
Create under 'Voortgang' on `/1/input`.

See design: https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=1195-25602&m=dev

### PR DESCRIPTION
**Pull request by lkleuver**
Adds progress bars on the data-entry home page showing percentages of first entry and completed polling stations

closes #242

### CODE REVIEW

**Review by jorisleker — CHANGES_REQUESTED** [review]
✅ design looks good.
✅ counting of 'alles samen' works as expected (counts percentage of polling stations with results
❌ counting of 'eerste invoer' (and probably 'tweede invoer') does not work as designed.

- 'Eerste invoer' should count the percentage with a *completed* first data entry as a percentage of the total number of polling stations. So this number should start at '0' and go to '100' when more polling station data comes in. As polling station data entries are deleted when the polling station is finalized, a polling station's first data entry should be counted as completed as follows: 
  - there is a record in `polling_station_data_entries` with (data.finalized=1 and entry_number=1) 
  - OR there is a record in `polling_station_results`

File: frontend/lib/ui/ProgressBar/ProgressBar.tsx
Comment (praseodym) [inline frontend/lib/ui/ProgressBar/ProgressBar.tsx:1]: Any reason not to use the [`<progress>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) element?

**Comment by lkleuver** [comment]
As discussed on vidconf, as long as there is only first_entry we are only showing the definitive progress bar for now.

**Review by jorisleker — APPROVED** [review]

File: frontend/lib/ui/ProgressBar/ProgressBar.tsx
Comment (lkleuver) [reply frontend/lib/ui/ProgressBar/ProgressBar.tsx:1]: I considered it but was worried about consistent styling.

**Review by praseodym — APPROVED** [review]

**Pull request merged**